docs/source/en/api/pipelines/stable_diffusion/control_net.mdx

```diff
@@ -0,0 +1,94 @@
+<!--Copyright 2023 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Text-to-Image Generation with ControlNet guidance
+
+## StableDiffusionControlNetPipeline
+
+ControlNet by [@lllyasviel](https://huggingface.co/lllyasviel) is a neural network structure to control diffusion models by adding extra conditions.
+
+It has integration with Stable Diffusion and 8 pre-trained models that conditions the models on different attributes 
+(such as edge detection, scribbles, depth maps, semantic segmentations and more)
```

Comment:
```suggestion
such as edge detection, scribbles, depth maps, semantic segmentations and more.

Using the pretrained models we can provide control images (for example, a depth map) to control Stable Diffusion text-to-image generation so that it follows the structure of the depth image and fills in the details.
```